### ISSUE TITLE
Delegation improvements

### ISSUE THREAD

**Issue by SpcCw** [friendica/friendica#8508]
Not sure if this is new feature or support request, maybe I just don't know how to do these things :)

I am trying out new delegation feature as I plan to have multiple accounts in different languages. So far I found these hindrances:

1. When selecting delegated account from the interface I get sent to .../profile/username page. Is there any use case for this? I think it would make more sense to open the default page (network). Why people would need to see their own profile by default? I think most of the time people are interested in reading their friends' feed so they have to click once again.

2. It would be very cool to make delegated accounts bookmarkable by URL (maybe by adding new parameter like ?delegate=username) so people with multiple accounts could go to them directly when they want to skipping the selection interface.

3. It would be nice to be able to switch accounts before interacting with a post. E.g. I speak English and Russian. When I use English account I often stumble upon a post in Russian (or vice versa) and I would rather interact with it using my other account however currently there is no easy way to do this. Another possible use case is when someone runs SFW and NSFW accounts. Using just whatever account can lead to unwanted effects such as making your friends from one account fetching your interactions intended for another account and getting their feed spammed with undesirable content.

**Comment by MrPetovan** [comment]
Would 3. be just a temporary switch for a specific interaction?

**Comment by SpcCw** [comment]
> Would 3. be just a temporary switch for a specific interaction?

Yes. So if someone has account "A" (active) and "B" (delegated) and they see something while browsing as "A" they can also interact "as B" just this time.

**Comment by SpcCw** [comment]
Also the first part of this feature request (_"When selecting delegated account from the interface I get sent to .../profile/username"_) was done some time ago, big thanks for that!
